src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1127,26 +1127,9 @@ class LeVADocumentUseCase extends DocGenUseCase {
     String createTRC(Map repo, Map data) {
         def documentType = DocumentType.TRC as String
 
-        def acceptanceTestData = data.tests.acceptance
-        def installationTestData = data.tests.installation
-        def integrationTestData = data.tests.integration
-
         def sections = this.getDocumentSections(documentType)
         def systemRequirements = this.project.getSystemRequirements()
 
-        // Compute the test issues we do not consider done (not successful)
-        def testIssues = systemRequirements.collect { it.getResolvedTests() }.flatten().unique().findAll {
-            [Project.TestType.ACCEPTANCE, Project.TestType.INSTALLATION, Project.TestType.INTEGRATION].contains(it.testType)
-        }
-
-        this.computeTestDiscrepancies(null, testIssues, junit.combineTestResults([acceptanceTestData.testResults, installationTestData.testResults, integrationTestData.testResults]))
-
-        def testIssuesWip = testIssues.findAll { !it.status.equalsIgnoreCase("cancelled") && (!it.isSuccess || it.isUnexecuted) }
-
-        def hasFailingTestIssues = !testIssuesWip.isEmpty()
-
-        def watermarkText = this.getWatermarkText(documentType, hasFailingTestIssues || this.project.hasWipJiraIssues())
```

Comment:
@s2oBCN please restore this line, without the `hasFailingTestIssues` and re-apply the watermark in line 1159.